### ISSUE TITLE
The results on Semantickitti testset are lower than mentioned in the paper

### ISSUE THREAD

**Issue by ytrhhiaya** [dvlab-research/SphereFormer#52]
I train the training set and validation set of semanticKITTI together, then I upload the test result to Codalab. But the mean IOU is 69.4, It is much lower than the 74.8mIOU showed in the paper. The gap is too large. I have chosen the model_last.pth. How did you choose the model during testing? Did you do any other processing during the testing? 
Thank you！

**Comment by illuosion** [comment]
I had the similiar result with u ,and my several classes' acc is nearly 0 ,what about u

**Comment by ytrhhiaya** [comment]
> I had the similiar result with u ,and my several classes' acc is nearly 0 ,what about u


Each class I measured has a corresponding IOU and acc, but the mean IOU is lower than claimed in the paper

**Comment by illuosion** [comment]
Could u please show me your results and environment by email illuosion0511@gmail.com

**Comment by X-Lai** [comment]
@WYRTDCQ  Hi, please refer to https://github.com/dvlab-research/SphereFormer/issues/1.

**Comment by hymhymhym123** [comment]
@illuosion hi,I have encountered the same problem as you. The miou of class7 and class11 is very low (no more than 0.01). Do you know why this is now
